code/ui/blocks/src/blocks/Description.tsx

```diff
@@ -43,58 +50,104 @@ const getInfo = (info?: Info) => info && (typeof info === 'string' ? info : str(
 
 const noDescription = (component?: Component): string | null => null;
 
-export const useDescriptionProps = (
-  { of, type, markdown, children }: DescriptionProps,
+const getDescriptionFromModuleExport = (of: DescriptionProps['of'] = 'meta'): string | null => {
+  const { projectAnnotations, ...resolvedModule } = useOf(of);
```

Comment:
I was expecting to see `useOf(of || 'component')` here. How does it work when there's no props passed? (I note there's no story for that either).

Reply:
it defaulted to `'meta'` at the function parameter. but since I've moved the useOf call, that is now `useOf(of || 'meta')` in the component.

The story for this is "Of Undefined Attached". https://635781f3500dd2c49e189caf-oinalxvggn.chromatic.com/?path=/story/storybook-blocks-blocks-description--of-undefined-attached
Is that unclear naming?

Reply:
Oh my bad! I missed that 👏